ORBIT/phases/design/mooring_system_design.py

```diff
@@ -209,7 +213,7 @@ def calculate_line_length_mass(self):
                 + self.rope_length * rope_mass_per_m
             ) / 1e3  # tonnes
 
-        elif self.mooring_type == "TLP":
+        elif self.mooring_type == "Tlp":
```

Comment:
I see how this is now annoying because there are many forms of standardization this could take.

Reply:
Yeah... Never a one size fits all, heh.